data/create-tables.js

```diff
@@ -17,7 +17,7 @@ async function run() {
                     email VARCHAR(256) NOT NULL,
                     hash VARCHAR(512) NOT NULL
                 );           
-                CREATE TABLE animals (
+                CREATE TABLE chords (
```

Comment:
Super close here! Looks like you updated the table name but also needed to update the properties to reflect your Chords columns instead of the animals data.